pkg/sdk/logging/model/output/azurestore.go

```diff
@@ -40,6 +40,9 @@ type _metaAzure interface{} //nolint:deadcode,unused
 type AzureStorage struct {
 	// Path prefix of the files on Azure
 	Path string `json:"path,omitempty"`
+	// Azure Cloud to use. e.g. AzurePublicCloud, AzureChinaCloud, AzureGermanCloud, AzureUSGovernmentCloud
+	// This field is supported only if the fluentd plugin honors it. e.g https://github.com/elsesiy/fluent-plugin-azure-storage-append-blob-lts
+	AzureCloud string `json:"azure_cloud,omitempty"`
```

Comment:
If we want to support Azure Stack Cloud, we may need to precise it should be in uppercase: `AZURESTACKCLOUD`

Reply:
Hey @aslafy-z , thanks for the comment. Do you mean adding it as an example in the comment ?

Reply:
I made a proposal :)